dependencies/pom.xml

```diff
@@ -517,16 +452,108 @@
                     </exclusion>
                 </exclusions>
             </dependency>
-            <dependency>
```

Comment:
done, the versions of core dependencies is set in `server/pom.xml`

Reply:
It is recommended to keep dependencies like mysql, Postgres, H2, mariadb directly in dependencies. pom. The dependency declaration in Dependencies is only dependencyManagement. Have we fully tested the above dependencies in Spring-boot? In fact, our application module does not directly rely on DB-driver dependencies such as mysql, Postgres, H2, Mariadb, etc. except server, but it can indicate which versions we recommend and adapt. Users may be able to manage dependencies using import seta-dependency.